indexer/workers/hist-queuer.py

```diff
@@ -29,26 +33,46 @@ def process_file(self, fname: str, fobj: BinaryIO) -> None:
         """
         called for each input file with open binary/bytes I/O object
         """
+        # try extracting date from file name to create fetch_date for RSSEntry.
+        m = DATE_RE.match(fname)
+        if m:
+            fetch_date = "-".join(m.groups())
+        else:
+            fetch_date = None
+
         # typical columns:
         # collect_date,stories_id,media_id,downloads_id,feeds_id,[language,]url
         for row in csv.DictReader(io.TextIOWrapper(fobj)):
             logger.debug("%r", row)
 
-            url = row.get("url", "")
+            url = row.get("url", None)
```

Comment:
```suggestion
            url = row.get("url")
```

Isn't `None` the default?